get-started/part2.md

```diff
@@ -82,16 +85,15 @@ ENV NAME World
 CMD ["python", "app.py"]
 ```
 
-This `Dockerfile` refers to a couple of things we haven't created yet, namely
-`app.py` and `requirements.txt`. Let's get those in place next.
+This `Dockerfile` refers to a couple of files we haven't created yet, namely
+`app.py` and `requirements.txt`. Let's create those next.
 
 ## The app itself
 
-Grab these two files and place them in the same folder as `Dockerfile`.
-This completes our app, which as you can see is quite simple. When the above
-`Dockerfile` is built into an image, `app.py` and `requirements.txt` will be
-present because of that `Dockerfile`'s `ADD` command, and the output from
-`app.py` will be accessible over HTTP thanks to the `EXPOSE` command.
+Create two more files, `requirements.txt` and `app.py`, and put them in
+the same folder with the `Dockerfile`. This will complete our app.
+
+The contents for these two new files is given below.
```

Comment:
I don't think deleting the `ADD` command's functionality is good here. Otherwise this is fine.